.vscode/settings.json

```diff
@@ -32,6 +32,10 @@
         "xlocmon": "cpp",
         "xlocnum": "cpp",
         "xloctime": "cpp",
-        "xmemory": "cpp"
+        "xmemory": "cpp",
```

Comment:
We should really delete this stupid file and add it to the .gitignore file in another pr😂

Reply:
Yes i really dont know why there are always changes. Although the conventionalCommits.scopes are also saved in here...